ai_presenter/voice_ai/base.py

```diff
@@ -61,6 +61,7 @@ def create_character_db(self, line: str):
                                                    self.actors[name].
                                                    description)
```

Comment:
it won't let me comment up there, but when the age converter method(defined in tools/age_converter.py) on line 56 throws an exception, the try block fails so it goes to the exception case and creates a male british voice